src/styles/blocks/_card.scss

```
@@ -0,0 +1,112 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $white-color;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    position: relative;
+    width: 100%;
+    padding-bottom: 80.72%;
+    &:not(:last-child) {
+      margin-bottom: 40px;
+    }
+    img {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      object-fit: contain;
+    }
+  }
+
+  &__content {
+    &:not(:last-child) {
+      margin-bottom: 16px;
+    }
+  }
+
+  &__title {
+    &:not(:last-child) {
+      margin-bottom: 4px;
+    }
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 1.4;
+    &:not(:last-child) {
+      margin-bottom: 16px;
+    }
+  }
+
+  &__rewievs {
+    &:not(:last-child) {
+      margin-bottom: 24px;
+    }
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__prices {
+    &:not(:last-child) {
+      margin-bottom: 16px;
+    }
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__button {
+    text-transform: uppercase;
+    display: flex;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $hover-color;
+    border: 1px solid $hover-color;
+    color: $white-color;
+    transition: all 0.4s;
+    @media (any-hover: hover) {
+      &:hover {
+        color: $hover-color;
+        background-color: $white-color;
+      }
+    }
```

it really cool that you used this media👍

add line break like in example below in all your code
```suggestion

    @media (any-hover: hover) {
      &:hover {
        color: $hover-color;
        background-color: $white-color;
      }
    }
```